userguide.md

```diff
@@ -0,0 +1,330 @@
+
+<center>
+<img src="/img/software/mapping_close.png" width="25%">
+<img src="/img/software/mapping_close_2.png" width="27%">
+<img src="/img/software/mapping_close_3.png" width="19%">
+</center>
+
+Remember, a good mapping is critical to the rest of the process. The user must be sure to get a good mapping before they start. They can revisit this mapping at any time if it needs to be corrected.
+
+Now that the user has computed a mapping, they can press the “Continue” button at the bottom right to begin the next step in the process: feature tracking.
+
+## Feature Tracking
+
+The next step in processing the video is feature-tracking. Feature-tracking is where the program tracks road users. When the program sees a moving object, it will notice “features” (points or edges) on this object, and it will follow them as the object moves throughout the video.
+
+The Feature-Tracking tab in the application is shown below:
+
+<center>
+<img src="/img/software/test_1.png" width="80%">
+</center>
+
+In order to properly track features for the video, the user will have to help the application fine-tune its feature-tracking algorithm. They can do this by modifying certain values called parameters. These parameters are listed on the right-hand portion of the screen. We have suggested some values for these parameters in the following steps.
+
+### **Step 1: Choose the number of frames to process in the test.**
+
+In order to test the parameter values on a short segment of the video first, the first two boxes on the right hand side of the application control which segment of the video the user will be testing. Descriptions of these two values are included below:
+
+<!-- \begin{table}[H]
+    \centering
+    \begin{tabular}{|p{0.36\textwidth}|p{0.54\textwidth}|}
+        \hline
+        Parameter Name and Sample Value &  Parameter Description \\
+        \hline
+        First frame to process. \newline
+        Starting value: 0 & First frame of the video to be processed. Despite the variable name, frames are indexed at 0 -- if the user wishes to start from the beginning of the video, they will set this parameter to 0. \\
+        \hline
+        Number of frames to process. \newline
+        Starting value: 200 & Number of frames to process. The user will set this so they can see at least a few seconds of video. They will be using this to test the other parameters later. \\
+        \hline
+    \end{tabular}
+\end{table}
+
+TABLE HERE -->
+
+### **Step 2: Set the feature-tracking parameters.**
+
+Now the user has to help the program figure out how best to track features. These parameters govern how the program tracks features, so this is a very important part of the process. The user can provide sample values to the program for each of these parameters. They must make sure to provide values for each parameter.
+
+The following parameters are related to feature-tracking:
+
+<!-- \begin{table}[H]
```

Comment:
Check this link out for creating a markdown table: http://www.tablesgenerator.com/markdown_tables